returnsrcvr-checker/src/main/java/org/checkerframework/checker/returnsrcvr/ReturnsRcvrAnnotatedTypeFactory.java

```diff
@@ -19,15 +22,21 @@
 import org.checkerframework.framework.type.typeannotator.TypeAnnotator;
 import org.checkerframework.javacutil.AnnotationBuilder;
 import org.checkerframework.javacutil.AnnotationUtils;
-import org.checkerframework.javacutil.TypesUtils;
 
 public class ReturnsRcvrAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
 
   AnnotationMirror THIS_ANNOT;
+  Collection<FrameworkSupport> frameworkSupports;
 
   public ReturnsRcvrAnnotatedTypeFactory(BaseTypeChecker checker) {
     super(checker);
     THIS_ANNOT = AnnotationBuilder.fromClass(elements, This.class);
+    
+    //add supports for frameworks
+    frameworkSupports = new ArrayList<FrameworkSupport>();
```

Comment:
@kelloggm instead of fixing this list here, we'd like to take in a list as a parameter to the constructor, or something equivalent.  Then, with a single command-line flag in the object-construction checker we could control the framework support there and in this checker.  However, it looks like the object construction checker currently instantiates this sub-checker via reflection.  I think we should go ahead and land this change once it's ready but we may need your help to figure out how to thread through the configuration properly

Reply:
I agree we should land this change as soon as it is ready, and deal with threading that through when we're ready.